test/dkg_commitmentmatrix_SUITE.erl

```diff
@@ -16,6 +24,12 @@ init_per_testcase(_, Config) ->
 end_per_testcase(_, Config) ->
     Config.
 
+new_matrix_test(Config) ->
+    Pairing = proplists:get_value(pairing, Config),
+    BiPoly = dkg_bipolynomial:generate(Pairing, 4),
+    io:format("BiPoly: ~p~n", [dkg_bipolynomial:print(BiPoly)]),
```

Comment:
What is this testing?

Reply:
Oh, this creeped in, I'll remove this test, my intention was to test the identity matrix, but can do it in another branch.